nodeList.json

```diff
@@ -2,51 +2,65 @@
   "nodeList": [
     {
       "altitude": 170,
-      "dateAdded": "2020-07-23T18:00:00.000Z",
+      "dateAdded": "2020-07-23",
```

Comment:
@YurkoWasHere we should change this to date only. The schema now validates it as such.

Reply:
sure, time is annoying to calculate anywa